src/Winton.Extensions.Configuration.Consul/ConsulConfigurationSource.cs

```diff
@@ -47,10 +44,12 @@ public ConsulConfigurationSource(string key, CancellationToken cancellationToken
 
         public bool ReloadOnChange { get; set; } = false;
 
+        public TimeSpan PollWaitTime { get; set; } = TimeSpan.FromMinutes(1);
```

Comment:
This property also needs to be added to `IConsulConfigurationSource` so it's available to the `AddConsul` callback.

Also, should it default to `5m` to preserve existing behavior, or are there reasons to reduce it to `1m`, which would surely put more load on the Consul server?

Reply:
Updated.